Xwt.XamMac/Xwt.Mac/MenuItemBackend.cs

```diff
@@ -149,6 +149,11 @@ public bool Checked
 			}
 		}
 
+		public void SetFormattedText (FormattedText text)
+		{
+			item.AttributedTitle = text.ToAttributedString ();
```

Comment:
Once set, it'll be always used, it's impossible to set an unformatted label afterwards.